sharedbuffers/mapped_struct.py

```diff
@@ -3581,6 +3799,24 @@ def _map_types(slot_types):
 
     @classmethod
     def from_typed_slots(cls, struct_class_or_slot_types, *p, **kw):
+        """
+        Constructs a :class:`Schema` out of a description of attribute (slot) types.
+
+        :param struct_class_or_slot_types: Either a class with a ``__slot_types__`` attribute,
+            or the mapping of attribute names to attribute types itself.
+
+        :keyword int alignment: *(default 8)* Enforce alignment on object sizes. Having objects
```

Comment:
Why using keyword and not param like in the other values?

Reply:
Basically it's a keyword-only argument (or, rather, only guaranteed to work through kwarg)